### DISCUSSION TITLE
Cant pair a device, but can send boost order from a backup

### DISCUSSION THREAD

**Discussion by Mig287** [OpenCCU/OpenCCU#2044] [🇺🇸 Q&A]
Hi Folks,

allready posted in german but maybe in english is better.

I have installed RaspberryMatic with an HMIP-RFUSB, on an Rasp4.
And cant manage to communicate with my devices properly. I took a Backup from my ccu3 and pasted it into RaspberryMatic (and to stop the "thats the problem shouting" I also reinstalled RaspberryMatic clean and tried this too) 
Properly means I only have Thermostats and I can send a "boost" order but neither can I change the temperature nor change from auto to manual Mode. 
I cant find any new devices, I  a device and tried.
TLDR:
- I cant communicate properly with my devices
- I cant connect any new devices
- I dont have the original Power Supply but tested two different Powerful ones 
- The HMIP-RFUSB is not at an USB 3 Port
- The HMIP-RFUSB is plugged into an extension cable
- Raspb 4
- I can post a log on demand.
- my carrier Sense is a 0% - edit, sometimes its at 1%
